docs/modules/ROOT/pages/subscriptions/mechanisms.adoc

```diff
@@ -0,0 +1,146 @@
+[[subscription-mechanisms]]
+= Subscription Mechanisms
```

Comment:
Could we use only "Mechanisms" as the title of the page or that would look to decontextualized?

Reply:
I felt it may be a bit confused as you say, but if you think otherwise I'm happy to change it